austation/code/modules/hydroponics/grown/towercap.dm

```diff
@@ -0,0 +1,9 @@
+/obj/item/seeds/tower/steel
+  name = "pack of steel-cap mycelium"
+  desc = "This mycelium grows into steel logs."
+  icon_state = "mycelium-steelcap"
+  species = "steelcap"
+  plantname = "Steel Caps"
+  product = /obj/item/grown/log/steel

```

Comment:
can you make this a real thing, that turns into metal sheets or some shit when cut, instead of a fake child object?